python/fledge/plugins/north/opcuaclient/client.py

```diff
@@ -187,13 +229,15 @@ async def check_server_reachability(self):
             _logger.debug("Successfully connected to {}".format(self.url))
             return True
         except Exception as e:
-            _logger.error("Failed to connect to {} - {}".format(self.url, str(e)))
+            _logger.debug("Failed to connect to {} - {}".format(self.url, str(e)))
```

Comment:
@ashish-jabble is this intentional?

Reply:
yupe, this is an internal function designed to prevent log duplication, as the logs primarily originate from the send_payloads method.